src/fu-util.c

```diff
@@ -2845,6 +2847,12 @@ main (int argc, char *argv[])
 	if (no_history)
 		priv->flags |= FWUPD_INSTALL_FLAG_NO_HISTORY;
 
+	/* start polkit tty agent to listen for password requests */
```

Comment:
don't you also want to check `is_interactive`?

Reply:
Yes! Fixed.